ports/stm32/boards/make-pins.py

```diff
@@ -1,59 +1,17 @@
 #!/usr/bin/env python
 
-"""
-Generates pin source files based on an MCU alternate-function definition (eg
-stm32f405_af.csv) and a board-specific pin definition file, pins.csv.
 
-The pins.csv file can contain empty lines, comments (a line beginning with "#")
```

Comment:
Would be good to keep this documentation about the format of pins.csv.  Maybe at the top of `tools/boardgen.py`?